README.md

```diff
@@ -1,28 +1,35 @@
 # AWSP - CLI To Manage your AWS Profiles!
-AWSP provide a interactive terminal to interact with your AWS Profile. The aim of this project is to make it easier to navigate, observe and manage your AWS Profile in the wild. 
+AWSP provides an interactive terminal to interact with your AWS Profiles. The aim of this project is to make it easier to navigate, observe and manage your AWS Profiles in the wild. 
 
 ---
-
+[![codecov](https://codecov.io/gh/kubeopsskills/awsp/branch/beta/graph/badge.svg?token=5VBFYN3BBH)](https://codecov.io/gh/kubeopsskills/awsp)
 [![release](https://img.shields.io/github/v/release/kubeopsskills/awsp?logo=awsp)](https://github.com/kubeopsskills/awsp/releases)
 [![License Apache](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/kubeopsskills/awsp/blob/beta/LICENSE-APACHE)
 [![License MIT](https://img.shields.io/badge/license-MIT-green?label=License)](https://github.com/kubeopsskills/awsp/blob/beta/LICENSE-MIT)
 [![Download](https://img.shields.io/github/downloads/kubeopsskills/awsp/total)](https://github.com/kubeopsskills/awsp/releases)
 
 ---
 
-
 ## Demo
 ![screenshot1](./assets/images/select-profile.png)
 ![screenshot2](./assets/images/select-region.png)
 
-## Prerequisite
+## Installation
+
+AWSP is available on Linux, macOS, macOS-Arm64 and Windows platforms in the [release](https://github.com/kubeopsskills/awsp/releases) page.
+- Binaries for Linux, macOS, and macOS-Arm64 are available as tarballs 
+- Binary for Windows is available as zip file
```

Comment:
>  Binary for Windows is available as zip file

Windows doesn't use binary for execution but it uses an executable file to execute it 

you should change like this `executable file for window ` ... `as a zip file`